src/boot/lzo.c

```diff
@@ -209,10 +209,5 @@ void Yaz0_Decompress(unsigned int pstart /* physical rom offset of file      */
         }
     }
 L_done:
-    do {
-    } while (0);
-#if MAJORA
-    dec.dst_end = op;
-    dec.buf_end = 0;
-#endif
-}
+    do { } while (0); }
```

Comment:
this do while can be removed right? like it does literally nothing from what I can tell